pyqmc/multislater.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+from slateruhf import sherman_morrison_row
+
+#Taken from autogenv2/pyscf2qwalk
+def binary_to_occ(S, ncore):
```

Comment:
need a docstring; what are the inputs/outputs of this function?

Reply:
handled